test/framework/src/main/java/org/opensearch/test/ParameterizedOpenSearchIntegTestCase.java

```diff
@@ -48,7 +48,8 @@ public void afterTests() {
     }
 
     public void indexRandomForConcurrentSearch(String... indices) throws InterruptedException {
-        if (dynamicSettings.get(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey()).equals("true")) {
+        if (dynamicSettings.get(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey()) != null
```

Comment:
```suggestion
       if ("true".equals(dynamicSettings.get(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey())))
```

Reply:
Alternatively we could use this and get rid of the `&&` altogether.
https://github.com/opensearch-project/OpenSearch/blob/c0852f84f0647f78a45e71396d6370a39ba4922f/server/src/main/java/org/opensearch/common/settings/Settings.java#L264-L271